packages/opentelemetry-core/src/trace/globaltracer-utils.ts

```diff
@@ -15,21 +15,31 @@
  */
 
 import * as types from '@opentelemetry/types';
-import { TracerDelegate } from './TracerDelegate';
+import { NoopTracerRegistry } from './NoopTracerRegistry';
 
-let globalTracerDelegate = new TracerDelegate();
+let globalTracerRegistry: types.TracerRegistry = new NoopTracerRegistry();
```

Comment:
A todo/ticket for later: delete the trace delegate code

Reply:
Is it not used/needed? I didn't really understand its purpose so I didn't want to remove it without a reason